partcad/src/partcad/converter.py

```
@@ -0,0 +1,169 @@
+
+        shape = CADConverter._read_shape(input_file, input_format)
+        CADConverter._write_shape(shape, output_file, output_format)
+
+    @staticmethod
+    def _read_shape(file_path: str, file_format: str) -> TopoDS_Shape:
+        """Read a CAD file and return its shape."""
+        file_format = file_format.lower()
+
+        if file_format == "step":
+            return CADConverter._read_step(file_path)
+        elif file_format == "stl":
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)